rascal-vscode-extension/src/lsp/RascalLSPConnection.ts

```diff
@@ -104,6 +111,27 @@ async function showContentPanel(url: string, title:string, viewColumn:integer):
     contentPanels.set(id, panel);
 }
 
+async function openEditor(uriString: string, range:vscode.Range, viewColumn: integer) {
+        const uri = vscode.Uri.parse(uriString);
+
+        const doc = await vscode.workspace.openTextDocument(uri);
+
+        // Show it in an editor
+        const editor = await vscode.window.showTextDocument(doc, {
+             // make sure it's not a preview, otherwise it will dissappear with the next focus change:
+            preview: false,
+            // put it where we want: if another editor is open for the same URI _and_ the same viewColumn, that one will be reused:
+            viewColumn: viewColumn,
```

Comment:
this does not implement the `-1` and `-2` features of rascal.

Reply:
yes it does. Why do you say that? That's copied straight from the VScode docs.

Reply:
```
export enum ViewColumn {
		/**
		 * A *symbolic* editor column representing the currently active column. This value
		 * can be used when opening editors, but the *resolved* {@link TextEditor.viewColumn viewColumn}-value
		 * of editors will always be `One`, `Two`, `Three`,... or `undefined` but never `Active`.
		 */
		Active = -1,
		/**
		 * A *symbolic* editor column representing the column to the side of the active one. This value
		 * can be used when opening editors, but the *resolved* {@link TextEditor.viewColumn viewColumn}-value
		 * of editors will always be `One`, `Two`, `Three`,... or `undefined` but never `Beside`.
		 */
		Beside = -2,
		/**
```